.github/workflows/linkcheck.yml

```diff
@@ -1,6 +1,7 @@
 name: LinkCheck - All files 🔗
 
 on:
+  pull_request:
```

Comment:
Added temporary to check if job runs fine

Reply:
The job output [here](https://github.com/pyvista/pyvista/actions/runs/14801773525/job/41561958259) is the same as in master branch [here](https://github.com/pyvista/pyvista/actions/runs/14688722200/job/41220938057)